spec/spec_helper.rb

```diff
@@ -0,0 +1,144 @@
+require 'simplecov'
+SimpleCov.start
+
+require 'factory_girl'
+require 'webmock'
+require 'webmock/rspec'
+require 'rspec'
+
+WebMock.disable_net_connect!(allow: 'codeclimate.com')
+
+HEADERS = {
+  'Accept' => 'application/vnd.blockscore+json;version=4',
+  'User-Agent' => 'blockscore-ruby/4.1.1 (https://github.com/BlockScore/blockscore-ruby)',
+  'Content-Type' => 'application/json'
+}
+
+require File.expand_path(File.join(File.dirname(__FILE__), '../spec/factories'))
+require File.expand_path(File.join(File.dirname(__FILE__), '../spec/support/response'))
+
+$LOAD_PATH.unshift(File.join(File.dirname(__FILE__), '..', 'lib'))
+$LOAD_PATH.unshift(File.dirname(__FILE__))
+
+def without_authentication
+  BlockScore.api_key = nil # clear API key
+end
+
+def with_authentication
+  BlockScore.api_key = 'sk_test_a1ed66cc16a7cbc9f262f51869da31b3'
+end
+
+# This file was generated by the `rspec --init` command. Conventionally, all
+# specs live under a `spec` directory, which RSpec adds to the `$LOAD_PATH`.
+# The generated `.rspec` file contains `--require spec_helper` which will cause
+# this file to always be loaded, without a need to explicitly require it in any
+# files.
+#
+# Given that it is always loaded, you are encouraged to keep this file as
+# light-weight as possible. Requiring heavyweight dependencies from this file
+# will add to the boot time of your test suite on EVERY test run, even for an
+# individual file that may not need all of that loaded. Instead, consider making
+# a separate helper file that requires the additional dependencies and performs
+# the additional setup, and require it from the spec files that actually need
+# it.
+#
+# The `.rspec` file also contains a few flags that are not defaults but that
+# users commonly want.
+#
+# See http://rubydoc.info/gems/rspec-core/RSpec/Core/Configuration
+RSpec.configure do |config|
+  config.include(FactoryGirl::Syntax::Methods)
+  config.include(WebMock::API)
+
+  config.before(:each) do
+    with_authentication
+
+    @api_stub = stub_request(:any, /.*api\.blockscore\.com\/.*/).
+      with(headers: HEADERS).
+      to_return do |request|
```

Comment:
Style/MultilineOperationIndentation: Align the operands of an expression in an assignment spanning multiple lines.